### PR TITLE
fix: auth2 pages now styled like login2

### PR DESCRIPTION
**Pull request by danielgent**
:partying_face: 

This is what it looks like now when running locally

image

*BUT* I'll have to update the snapshots only when this is merged, as in the playwright snapshot helper I'm doing 

```
    const responseBody = responseText.replace(
      "/css/tailwind.css",
      "http://auth2.sesamy.dev/css/tailwind.css",
    );
```

Which will only be updated *after* merging this of course

It's not ideal *BUT* after this we shouldn't have to make any major tailwind CSS changes... so there should only be a few cases of this.

### CODE REVIEW

File: scripts/build-tailwind-css.sh [hunk 1]
```diff
@@ -1,3 +1,3 @@
 yarn postcss
 node build-tailwind
-yarn format # could be more targeted and just do tailwind.ts
\ No newline at end of file
+npx prettier --write src/styles/tailwind.ts
```
Comment (danielgent) [inline scripts/build-tailwind-css.sh:3]: quicker than doing them all

File: src/app.ts [hunk 2]
```diff
@@ -140,7 +140,7 @@ app.get(
     const css = tailwindCss;
 
     return ctx.text(css, 200, {
-      "content-type": "text/css",
+      "content-type": "text/css; charset=utf-8",
```
Comment (danielgent) [inline src/app.ts:143]: I could revert this... I thought this might stop the deencoding but of course not

File: src/styles/tailwind.ts [hunk 3]
```diff
@@ -901,6 +905,27 @@ h6{
...
+  align-content: center;
+  align-items: center;
+  justify-content: center;
+}
+.column-left{
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  justify-content: center;
+}
```
Comment (danielgent) [inline src/styles/tailwind.ts:928]: I've put Kim's library back as we were using the classes

File: src/styles/tailwind.ts [hunk 4]
```diff
@@ -771,19 +771,19 @@ h6{
 .max-h-full{
   max-height: 100%;
 }
-.min-h-\[calc\(100vh-83px\)\]{
+.min-h-\\[calc\\(100vh-83px\\)\\]{
```
Comment (danielgent) [inline src/styles/tailwind.ts:774]: double encoding in action :smile:

File: tailwind.config.js [hunk 5]
```diff
@@ -1,4 +1,7 @@
+const tailwindRowsColumns = require("@ape-egg/tailwind-rows-columns");
```
Comment (danielgent) [inline tailwind.config.js:1]: Kim's plugin was needed after all

File: src/utils/components/Layout.tsx [hunk 6]
```diff
@@ -91,7 +91,7 @@ const Layout: FC<LayoutProps> = ({ title, children, vendorSettings }) => {
 
       <body>
         <div
-          class="row min-h-full w-full overflow-hidden bg-cover bg-center text-sm sm:bg-fixed sm:bg-left-top"
+          class="row min-h-full w-full overflow-hidden bg-cover bg-center text-sm sm:bg-fixed sm:bg-left-top sm:pt-16 py-2"
```
Comment (danielgent) [inline src/utils/components/Layout.tsx:94]: Once this is merged I'll do another round of styling to make sure there are no more missing classes

**Review by markusahlstrand — APPROVED** [review]
Looks good. Check out if this would be worth using?
https://hono.dev/helpers/css

File: src/app.ts (hunk 2 above)
Comment (danielgent) [reply src/app.ts:143]: I don't see that, I see `css` in backticks https://hono.dev/helpers/css#css

It is marked `experimental` though

Comment (danielgent) [reply src/app.ts:143]: https://hono.dev/api/context has no `css()` part of the doc...

I think this whole PR could be replaced with that CSS helper though... without needing the double encoding. I'll try that now

Comment (markusahlstrand) [reply src/app.ts:143]: Experimental just means that it's reeeeaaaaly cool!

**Comment by danielgent** [comment]
I'll merge this for now @markusahlstrand and we can try out the Hono CSS helper on https://github.com/sesamyab/auth/pull/768

**Pull request merged**